app/src/main/res/xml/shortcuts.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+    <shortcut
+        android:shortcutId="switch_mode"
+        android:enabled="true"
+        android:icon="@drawable/ic_outline_next_24dp"
+        android:shortcutShortLabel="@string/shortcut_switch_mode">
+        <intent
+            android:action="android.intent.action.VIEW"
+            android:targetPackage="ru.karasevm.privatednstoggle"
+            android:targetClass="ru.karasevm.privatednstoggle.ui.SettingsDialogActivity"
+            android:data="myapp://switch_mode"/>
```

Comment:
The placeholder `myapp://` scheme should be replaced with with `privatednstoggle://` as that seems to be the common practice.